src/RadixDlt.NetworkGateway.PostgresIntegration/DbQueryExtensions.cs

```diff
@@ -89,7 +99,12 @@ public static IQueryable<LedgerTransaction> GetLatestLedgerTransactionBeforeStat
             .Take(1);
     }
 
-    public static IQueryable<LedgerTransaction> GetFirstLedgerTransactionAfterStateVersion<TDbContext>(this TDbContext dbContext, long afterStateVersion)
+    /// <summary>
+    /// The long/horrible/verbose name is on purpose, to draw your attention and prevent people foot-gunning themselves.
+    /// A LedgerTransaction row contains large blobs, so you must SELECT the fields you need after using this, and not pull down the whole
+    /// ledger transaction row, to avoid possible performance issues.
+    /// </summary>
+    public static IQueryable<LedgerTransaction> GetFirstLedgerTransactionAfterStateVersion__EnsureYouSelectRequiredFieldsAsRowsCanBeVeryBig<TDbContext>(this TDbContext dbContext, long afterStateVersion)
```

Comment:
this extension is used only in one place, maybe better to just inline it in plece where it's used?